rocAL_pybind/examples/rocAL_api_audio_unittest.py

```diff
@@ -0,0 +1,76 @@
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import random
+import numpy as np
+from amd.rocal.plugin.pytorch import ROCALAudioIterator
+import torch
+# torch.set_printoptions(threshold=10_000)
+np.set_printoptions(threshold=1000, edgeitems=10000)
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import math
+# import rocal_pybind.tensor
+import sys
+import cv2
```

Comment:
Please remove all cv2 imports as it is not used

Reply:
done